<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HeightmapWidget</class>
 <widget class="QDialog" name="HeightmapWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>303</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Heightmap</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">font: 75 9pt &quot;MS Shell Dlg 2&quot;;
font-weight:bold;</string>
       </property>
       <property name="text">
        <string>Import:</string>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_2">
       <property name="styleSheet">
        <string notr="true">font: 75 9pt &quot;MS Shell Dlg 2&quot;;
font-weight:bold;</string>
       </property>
       <property name="text">
        <string>Export:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="importButton">
       <property name="text">
        <string>Import</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="label_3">
       <property name="styleSheet">
        <string notr="true">font: 75 9pt &quot;MS Shell Dlg 2&quot;;
font-weight:bold;</string>
       </property>
       <property name="text">
        <string>Settings:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_4">
       <property name="styleSheet">
        <string notr="true">margin-left:25px;</string>
       </property>
       <property name="text">
        <string>File:</string>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="label_6">
       <property name="styleSheet">
        <string notr="true">margin-left:25px;</string>
       </property>
       <property name="text">
        <string>Scale:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="exportFileBox"/>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="importFileBox"/>
     </item>
     <item row="10" column="1">
      <widget class="QDoubleSpinBox" name="heightmapScale">
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::UpDownArrows</enum>
       </property>
       <property name="maximum">
        <double>10.000000000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_5">
       <property name="styleSheet">
        <string notr="true">margin-left:25px;</string>
       </property>
       <property name="text">
        <string>File:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QDoubleSpinBox" name="exportScaleBox">
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::UpDownArrows</enum>
       </property>
       <property name="minimum">
        <double>0.100000000000000</double>
       </property>
       <property name="maximum">
        <double>10.000000000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="importBrowseButton">
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="importScaleBox">
       <property name="minimum">
        <double>0.100000000000000</double>
       </property>
       <property name="maximum">
        <double>10.000000000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="QPushButton" name="exportBrowseButton">
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_7">
       <property name="styleSheet">
        <string notr="true">margin-left:25px;</string>
       </property>
       <property name="text">
        <string>Scale:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_8">
       <property name="styleSheet">
        <string notr="true">margin-left:25px;</string>
       </property>
       <property name="text">
        <string>Scale:</string>
       </property>
      </widget>
     </item>
     <item row="8" column="2">
      <widget class="QPushButton" name="exportButton">
       <property name="text">
        <string>Export</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>HeightmapWidget</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>HeightmapWidget</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
